packages/backend-api/integration/app/AppKernel.php.patch

```diff
@@ -0,0 +1,38 @@
+@@ -19,8 +19,10 @@
+             new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
+             new Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle(),
+             new FM\ElfinderBundle\FMElfinderBundle(),
++            new FOS\RestBundle\FOSRestBundle(),
```

Comment:
Could make problems... yes. Could you please test it? How much problems? Because the `patch` program is pretty clever :-)

We also sort bundles alphabetically, that's why it is here.

What is more important - keeping our standards or making the integration less pain?